webapp/src/routes/InfoPage/InfoPageMobile.js

```diff
@@ -340,13 +325,27 @@ const InfoPageMobile = () => {
     }
   }
 
+  useEffect(() => {
+    getInfo()
+  }, [getInfo, location])
+
+  const getOffers = async () => {
+    if(profile){
+      if(profile.role === 'sponsor'){
+      setLoadingOffers(true)
+      await getAllOffers()
```

Comment:
Check the indentation

Reply:
@JustinCast just solved, just to remember, the purpose of this PR is to solve the console warnings, not indentations or similar errors coming from others merge changes.

Reply:
Ok, got it. Good job